src/main/scala/io/iohk/ethereum/jsonrpc/EthService.scala

```diff
@@ -19,7 +19,7 @@ import io.iohk.ethereum.transactions.PendingTransactionsManager
 import scala.concurrent.Future
 import scala.util.{Failure, Success, Try}
 
-
+// scalastyle:off number.of.methods number.of.types
```

Comment:
maybe we should split EthService into logical parts?
for example: mining and blockchain